### PR TITLE
Adding GitHub identifier prefix

### PR DESCRIPTION
**Pull request by nestorbonilla**
### Description

Including GitHub off-chain prefix identifier to allow mapping of usernames of this repository hosting with addresses in SocialConnect.

### Other changes

None

### Tested

None

### Related issues

None

### CODE REVIEW

**Review by alecps — APPROVED** [review]
thanks!

File: packages/sdk/base/src/identifier.ts [hunk 1]
```diff
@@ -38,6 +38,7 @@ export enum IdentifierPrefix {
   DISCORD = 'discord',
   TELEGRAM = 'telegram',
   SIGNAL = 'signal',
+  GITHUB = 'github'
```
Comment (alecps) [inline packages/sdk/base/src/identifier.ts]: maybe lint check is failing bc you need a comma here?

Comment (nestorbonilla) [reply packages/sdk/base/src/identifier.ts]: added on a new commit

**Comment by aaronmgdr** [comment]
These Identifiers should be moved into a package in the social-connect repo. base package is to broad for them

**Comment by aaronmgdr** [comment]
closing. added changes to https://github.com/celo-org/social-connect/pull/104

**Pull request closed without merging**